packages/jasmine/src/adapter/JasmineConfig.ts

```diff
@@ -81,4 +81,14 @@ export interface JasmineConfig {
      * @see https://jasmine.github.io/api/edge/jasmine#.DEFAULT_TIMEOUT_INTERVAL
      */
     defaultTimeoutInterval?: number;
+
+    /**
+     * @desc
+     *  A list of additional reporters which will be added to the test runner.<BR>
+     *  Useful for situations like configuring ReportPortal, because you cannot do `jasmine.addReporter()` in the protractor config.
+     *  Note: reporters must be instantiated before adding them to the configuration.
+     *
+     * @type {string[] | undefined}
+     */
+    additionalReporters?: string[];
```

Comment:
I'd suggest calling this option `reporters` instead of `additionalReporters` to keep it simple